grails-app/services/au/org/ala/collectory/SitemapService.groovy

```diff
@@ -106,22 +112,23 @@ class SitemapService {
         countUrls++
     }
 
+    @Transactional
     def buildSitemap() throws Exception {
 
         Collection.findAll().each {Collection it ->
-            writeUrl(it.lastUpdated, "weekly", grailsApplication.config.grails.serverURL + "/public/show/co" + it.id)
+            writeUrl(it.lastUpdated, "weekly", grailsApplication.config.grails.serverURL + "/public/show/" + it.uid)
```

Comment:
Should be `uid` and not `id`. `SELECT id, uid FROM collection order by 1;` will show that they are not the same (at least not in a database where items have been deleted).